cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -33,33 +33,49 @@ import org.apache.spark.sql.{Row, SaveMode, SnappyContext}
  */
 class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
 
-  val currenyLocatorPort = ClusterManagerTestBase.locPort
+  val currentLocatorPort = locPort
 
   def testTableCreation(): Unit = {
     startSparkJob()
   }
 
+  //After discussing with Namratam Commenting out this test as it causes other tests to fail
+/*
+
   def testTableCreationWithHA(): Unit = {
```

Comment:
Has this test been added recently? I didn't see failures due to this test previously.
And its Namrata not Namratam :-)